apps/client/src/app/components/admin-market-data/create-asset-profile-dialog/create-asset-profile-dialog.component.ts

```diff
@@ -181,6 +201,22 @@ export class GfCreateAssetProfileDialogComponent implements OnInit, OnDestroy {
       });
   }
 
+  private initializeExchangeRateDataSource() {
+    this.adminService
+      .fetchAdminData()
```

Comment:
Instead of fetching admin data twice, rename `initializeCustomCurrencies()` to `initialize()` and initialize the exchange rate data source there.